### PR TITLE
Improvement/artesca 11213 bucket creation tooltip

### PR DESCRIPTION
**Pull request by JeanMarcMilletScality**

### CODE REVIEW

File: src/react/workflow/ReplicationForm.tsx [hunk 1]
```diff
@@ -286,14 +286,34 @@ function ReplicationForm({ prefix = '', isCreateMode, ...props }: Props) {
...
-                            ?.supportsReplicationSource ||
-                          !isBucketVersionned ||
-                          isVeeamBucket
-                        }
+                        }) => ({
+                          disabled:
+                            isReplicationSourceSupported ||
+                            !isBucketVersionned ||
+                            isVeeamBucket,
+                          disabledMessage: isReplicationSourceSupported ? (
```
Comment (JBWatenbergScality) [inline src/react/workflow/ReplicationForm.tsx:296]: Can we extend the existing workflow tests to check the presence of the tooltip ?

File: src/react/workflow/SourceBucketOption.tsx [hunk 2]
```diff
@@ -17,9 +16,9 @@ import { useEffect } from 'react';
 type DisableOptionProps = {
   disableOption?: (obj: {
     isBucketVersionned?: boolean;
-    locationInfos?: LocationStorageInfos;
+    isReplicationSourceSupported?: boolean;
     isVeeamBucket: boolean;
-  }) => boolean;
+  }) => { disabled: boolean; disabledMessage?: JSX.Element };
```
Comment (hervedombya) [inline src/react/workflow/SourceBucketOption.tsx:21]: type `ReactNode` is better

Comment (JeanMarcMilletScality) [reply src/react/workflow/SourceBucketOption.tsx:21]: We only want a string or an ul element. Maybe we should use a union type ?

**Review by JBWatenbergScality — APPROVED** [review]
LGTM !

**Pull request merged**